tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = yapf-check,mypy,lint,isort-check,unit
+envlist = yapf-inplace,mypy,lint
```

Comment:
no `unit` when you run `tox`?

Reply:
Just for now, didn't see unit doing anything